src/popup/router/components/Modal.vue

```diff
@@ -201,9 +201,10 @@ export default {
       padding: 8px var(--screen-padding-x);
       background:
         linear-gradient(
-          0deg,
-          rgba(var(--screen-bg-color), 0.9) 50%,
-          rgba(var(--screen-bg-color), 0) 100%
+          180deg,
+          rgba(25, 25, 25, 0) 0%,
+          rgba(25, 25, 25, 0.8) 43.08%,
+          rgba(25, 25, 25, 0.9) 90.79%
```

Comment:
First problem I see here is the direct use of colors. We should take them from SCSS variables.
The second and bigger problem is that I tweaked this part of the Modal in my PR here: https://github.com/aeternity/superhero-wallet/pull/1601/files#diff-81f7e7dfe274e46405414664589d76fde15fe58185d5e85eb32372ad60a67bb9R206